Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,13 @@
 import React from 'react';
+import CardBanner from './CardBanner';
+import CardContent from './CardContent';
 import './Card.css';
```

Comment:
General rule of thumb is only capitalize React Components.. this is to keep the confusion down when you have a lot of files and see something like CardContainer.css and CardContainer.js. Just makes it easier to read.